lib/trino-filesystem-gcs/src/main/java/io/trino/filesystem/gcs/GcsFileSystem.java

```diff
@@ -169,8 +169,8 @@ public void deleteFiles(Collection<Location> locations)
             for (List<Location> locationBatch : partition(locations, batchSize)) {
                 StorageBatch batch = storage.batch();
                 for (Location location : locationBatch) {
-                    getBlob(storage, new GcsLocation(location))
```

Comment:
I'm curious, did you spot it by chance or was there a delete files in bulk slow?